backend/capellacollab/core/database/__main__.py

```diff
@@ -40,6 +42,7 @@ def migrate_db():
 
         context = MigrationContext.configure(conn)
         current_rev = context.get_current_revision()
+        tools_exist = inspect(engine).has_table('tools')
```

Comment:
Because `tools_exist` is used only once, I would suggest not to create it, i.e.: <br>
`if not inspect(engine).has_table('tools'):` <br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; `create_tools()`

Reply:
It’s not possible. The condition must be evaluated before a possible creation of the tools table, but the insertion of the tools must be executed after such creation.